pkgs/by-name/ai/aider/package.nix

```
@@ -0,0 +1,104 @@
+{
+  lib,
+  python311,
+  fetchFromGitHub,
+  git,
+  portaudio,
+}:
+
+let
+  python3 = python311.override {
+    self = python3;
+    packageOverrides = _: super: { tree-sitter = super.tree-sitter0_21; };
```

```suggestion
    packageOverrides = _: super: { tree-sitter = super.tree-sitter_0_21; };
```

due to https://github.com/NixOS/nixpkgs/pull/332747/commits/345473954a782869d97be41ee2723223aae9d274